# Service to get the cartesian position of the robot
uint8 frameId
---
float64 x
float64 y
float64 z
float64 q0
float64 qx
float64 qy
float64 qz
uint8 frameId
int64 ret
string msg